plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TableStatisticsWriter.java

```diff
@@ -219,28 +213,33 @@ private CollectedStatistics mergeStatisticsIfNecessary(
                             List<BlobMetadata> toRead = reader.fileMetadata().blobs().stream()
                                     .filter(blobMetadata -> blobMetadata.type().equals(APACHE_DATASKETCHES_THETA_V1))
                                     .filter(blobMetadata -> blobMetadata.inputFields().size() == 1)
-                                    .filter(blobMetadata -> pendingPreviousNdvSketches.contains(getOnlyElement(blobMetadata.inputFields())))
```

Comment:
this should be same as `hasUsefulData` above, i.e.

```
.filter(blobMetadata -> columnsWithRecentlyComputedStats.contains(getOnlyElement(blobMetadata.fields())))
```

Reply:
I probably lost it while changing the implementation few times. Thanks for catching it